src/components/templates/PartyMap/components/RoomAttendance.tsx

```diff
@@ -18,14 +18,19 @@ const MAX_AVATARS_VISIBLE = 2;
 
 export const RoomAttendance: FC<PropsType> = ({ venue, room }) => {
   const partygoers = useSelector(partygoersSelector);
+
   const usersInRoom =
     partygoers?.filter(
       (partygoer) => partygoer.lastSeenIn[`${venue.name}/${room.title}`]
     ) ?? [];
+
   const numberOfUsersInRoom = usersInRoom?.length;
+
   if (numberOfUsersInRoom < 1) return <></>;
+
   return (
     <div className="attendance-avatars">
+      {/* TODO: extract this + useMemo */}
       {usersInRoom.map((user, index) => {
         return (
           <div key={`user-avatar-${index}`}>
```

Comment:
- [ ] [TODO] extract this + `useMemo`

Reply:
resolved in https://github.com/sparkletown/sparkle/pull/840/commits/881676de95fe02cb743898b3bff51818292514fa